src/routes.ts

```diff
@@ -181,6 +183,11 @@ export abstract class Routes {
           since: params?.since ?? new Date(0).toISOString(),
         },
       }),
+    get: (params?: EventsGetRequest) =>
```

Comment:
aren't parameters always required?

Reply:
I just noticed this. For this method, it should be removed (which I'll do rn), but the list method above has them typed as optional.